specs/backend-api.graphql

```diff
@@ -99,9 +110,79 @@ type Region {
   regionIdentifier: String!
 }
 
+type SignInPayload {
+  token: String!
+  user: Administrator!
+}
+
+enum AmountOfWorkUnit {
+  HOURS_PER_WEEK
+  HOURS_PER_YEAR
+}
+
+enum ApplicationType {
+  FIRST_APPLICATION
+  RENEWAL_APPLICATION
+}
+
+enum BlueCardEntitlementType {
+  JULEICA
+  SERVICE
+  STANDARD
+}
+
+enum BlueCardServiceEntitlementActivity {
+  DISASTER_CONTROL
+  FIRE_DEPARTMENT
+  RESCUE_SERVICE
+}
+
+enum GoldenCardEntitlementType {
+  HONOR_BY_MINISTER_PRESIDENT
+  SERVICE_AWARD
+  STANDARD
+}
+
 "Long type"
 scalar Long
 
+"A file part in a multipart request"
+scalar Upload
+
+input AttachmentInput {
+  data: Upload!
+  fileName: String!
+}
+
+input AuthDataInput {
+  email: String!
+  password: String!
+}
+
+input BlueCardApplicationInput {
+  applicationType: ApplicationType!
+  entitlement: BlueCardEntitlementInput!
+  givenInformationIsCorrectAndComplete: Boolean!
+  hasAcceptedPrivacyPolicy: Boolean!
+  personalData: PersonalDataInput!
+}
+
+"""
+Entitlement for blue EAK.
+    Either entitlementType == Juleica and juleicaNumber, juleicaExpirationDate, copyOfJuleica are not null
```

Comment:
What is this Juleica?

Reply:
Is this maybe a little bit too specific?

Reply:
It's the Jugendleitercard and yet another way to get an eak.
Please check back with the formulas if you think this is too specific. I think it's fine, as this is one of the three ways to get a blue card.